lib/constants/config.js

```diff
@@ -1,2 +1,6 @@
 export const CONFIG_LOAD = 'CONFIG_LOAD';
 export const CONFIG_RELOAD = 'CONFIG_RELOAD';
+
+export const BELL_STYLE = {
```

Comment:
Put this directly in the fil. it's not reused.

Reply:
@ppot You are right, currently it is not being reused anywhere else. The only thought I had was "maybe" there would be a time where other options would be available, such as "VISUAL" for a visual bell style. It seems out of scope for this iteration so we can remove this and have the condition that is using this just hardcode the string in the conditional instead of referencing this object enum.

Reply:
@ppot You are right, currently it is not being reused anywhere else. The only thought I had was "maybe" there would be a time where other options would be available, such as "VISUAL" for a visual bell style. It seems out of scope for this iteration so we can remove this and have the condition that is using this just hardcode the string in the conditional instead of referencing this object enum.